src/overlays/actors/ovl_En_Bji_01/z_en_bji_01.h

```diff
@@ -2,12 +2,31 @@
 #define Z_EN_BJI_01_H
 
 #include <global.h>
+#include "../ovl_Obj_Moon_Stone/z_obj_moon_stone.h"
 
 struct EnBji01;
 
+typedef void (*EnBji01ActionFunc)(struct EnBji01*, GlobalContext*);
+
 typedef struct EnBji01 {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x170];
+    /* 0x144 */ EnBji01ActionFunc actionFunc;
+    /* 0x148 */ SkelAnime skelAnime;
+    /* 0x18C */ ColliderCylinder collider;
+    /* 0x1D8 */ Vec3s jointTable[0x10]; 
```

Comment:
We generally have the size of these in decimal

Reply:
Will be fixed in a batch commit.